bedrock/firefox/templates/firefox/mobile-en.html

```diff
@@ -0,0 +1,219 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+{% from "macros-protocol.html" import feature_card, hero, call_out_compact with context %}
+
+{% add_lang_files "firefox/mobile-2019" %}
+{% add_lang_files "firefox/sendto" %}
```

Comment:
This should appear only once. To include multiple lang files you can list them in series in a single block:

`{% add_lang_files "firefox/mobile-2019" "firefox/sendto" %}`